### ISSUE TITLE
[Bug]: [Crash]: Super Factory Manager : Printing Press

### ISSUE THREAD

**Issue by eniwho** [AllTheMods/ATM-10#1347]
### Possible Fixes

Yes

### Modpack Version

2.0

### What happened?

When using the printing press from super factory manager and a printing form of a protection VIII book the press will make one book and then when I attempt to print another book it does not use the ingredients to make the book and when I attempt to take them off of the press it crashes the game.

**Comment by nk34player** [comment]
Yes it does happen, also checking mod's author he mentioned it will be fixed in the next version hopefully (by looking at the project to-do)

**Comment by JoyfulEngineer** [comment]
10-2.20 still has this issue.  I've also successfully made 8 copies, then had all but one disappear when I try to move the first one in my inventory.  This has happened both moving to the backpack and moving to an anvil trying to combine.  Game crash message refers to a server communication failure though I'm playing a single player world not open to others.  I receive the same crash message when interacting with either the SFM printing press or the Library after making a copy.
![Screenshot 2025-01-09 001600](https://github.com/user-attachments/assets/52a28dd8-1d27-4643-a6d8-49b327123dc4)
![Screenshot 2025-01-09 005325](https://github.com/user-attachments/assets/0745dd9b-0332-447a-9146-ee9d719b8e8b)
